hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableOutputFormat.java

```diff
@@ -76,7 +76,7 @@
 // TODO Move dictionary generator which is coded in spark to MR framework.
 public class CarbonTableOutputFormat extends FileOutputFormat<NullWritable, ObjectArrayWritable> {
 
-  protected static final String LOAD_MODEL = "mapreduce.carbontable.load.model";
+  public static final String LOAD_MODEL = "mapreduce.carbontable.load.model";
```

Comment:
please move all public variable together into the beginning of this class

Reply:
done